LibCpp2IL/Metadata/Il2CppGenericContainer.cs

```diff
@@ -16,6 +16,8 @@ public class Il2CppGenericContainer : ReadableClass
     /* Our type parameters. */
     public int genericParameterStart;
 
+    private bool IsOwnedByMethod => isGenericMethod != 0;
```

Comment:
Honestly it's probably better to just define the field as a bool and change Read to do the `!= 0` check